peachjam/admin.py

```diff
@@ -340,7 +378,7 @@ class JudgmentMediaSummaryFileInline(BaseAttachmentFileInline):
 
 
 class JudgmentAdminForm(DocumentForm):
-    hearing_date = forms.DateField(widget=forms.SelectDateWidget(), required=False)
+    hearing_date = forms.DateField(widget=DateSelectorWidget())
```

Comment:
I think this is still optional, and the required=False is still necessary?

Reply:
Right. changed